### ISSUE TITLE
Accessibility: Color contrast insufficient [Featured Product]

### ISSUE THREAD

**Issue by vikrampm1** [woothemes/woocommerce#52139]
**Prerequisites (mark completed items with an [x]):**
- [x] I have checked that my issue type is not listed here https://github.com/woocommerce/woocommerce/issues/new/choose
- [x] My issue is not a security issue, support request, bug report, enhancement or feature request (Please use the link above if it is).

### Issue Description:

The overlay on this background has an opacity that is too light which results in a color contrast failure for the smaller description text if the product or category image is very light. In the featured product, for example, the failures only occur in certain places, but this could cause a readbility issue particularly on websites that have photographs of products on white backgrounds.

![Image](https://github.com/user-attachments/assets/a8eb60b9-6402-4951-bedf-7154007dd377)

![Image](https://github.com/user-attachments/assets/84263f83-de6e-4675-aa82-6e5210b295e0)

### Component or Page: 

Component: Featured Product Block 

### Name affected element:

Description text for the featured category or featured product block

### Relevant code snippets:

```
.wp-block-woocommerce-featured-category {
    background-color: transparent;
    border-color: transparent;
    box-sizing: border-box;
    color: #fff;
}

<div class=""background-dim__overlay"" style=""background-color: #000000""></div>
```

### Recommended fix:

Change the opacity of the background overlay to be darker so there is at least 4.5:1 color contrast for the description text if an image that is largely white is placed behind it.

**Comment by Manussakis** [comment]
The Feature Product block provides editors with tools that allow changing the background color and the opacity value of the overlay.
Editors can achieve the best combination depending on the image to make the color contrast between the text accessible.

![Image](https://github.com/user-attachments/assets/a614eef4-a3d3-4769-8898-1d9a4fa15a61)

This issue can be closed since there is no further work to do.

cc: @vikrampm1
